docs/types/mapping-types.rst

```diff
@@ -4,7 +4,7 @@
 Mapping Types
 =============
 
-You declare mapping types with the syntax ``mapping(_KeyType => _ValueType)``.
+You declare mapping types with the syntax ``mapping(_KeyType => _ValueType) _VariableModifiers _VariableName``.
```

Comment:
This is how you declare a variable of mapping type, not how you declare a mapping type.

Maybe `declare` is also the wrong word, `use` may be better.